wiki/Crave_Rules.md

```diff
@@ -3,7 +3,7 @@ These are some rules users are expected to follow. Ignoring these rules may resu
 ## Devspace CLI Rules
 - ***Do not use repo sync, use crave run instead***. If you try to install repo tool manually, it is looked at as an act to circumvent the rules and will likely result in a ban
 - ***Be mindful of Devspace CLI Storage***. Clean up your .zip and .img build artefacts if you're used to pulling them to devspace
-- ***Do not rm -rf a crave clone***. Use crave clone destroy instead.
+- ***Do not rm -rf a crave clone***. Use crave clone destroy foldername instead.

```

Comment:
/path/to/folder makes more sense because that shows it is a path
What do you think?

Lgtm otherwise

Reply:
or something like path/to/folder/

Reply:
I agree

Reply:
changed